modules/gdnative/godot.cpp

```diff
@@ -35,6 +35,7 @@
 #include "global_config.h"
 #include "global_constants.h"
 #include "variant.h"
+#include "os/os.h"
```

Comment:
We should put a link to the contributing docs: http://docs.godotengine.org/en/latest/community/contributing/index.html. There it mentions the PR workflow and the clang-format usage.